api/preprints/views.py

```diff
@@ -416,6 +416,27 @@ def get_serializer_context(self):
         return context
 
 
+class PreprintBibliographicContributorsList(PreprintContributorsList, generics.ListAPIView):
+    permission_classes = (
+        AdminOrPublic,
+        drf_permissions.IsAuthenticatedOrReadOnly,
+        base_permissions.TokenHasScope,
+    )
+
+    pagination_class = PreprintContributorPagination
+    serializer_class = PreprintContributorsSerializer
+
+    view_category = 'preprints'
+    view_name = 'preprint-bibliographic-contributors'
+
+    def get_default_queryset(self):
+        contributors = super().get_default_queryset()
+        return contributors.filter(visible=True)
+
+    def post(self, request, *args, **kwargs):
+        raise MethodNotAllowed(method=request.method)
```

Comment:
I'm surprised this is necessary. shouldn't this be achived via inheirtance?

Reply:
I have it inheriting from `PreprintContributorsList ` (Which inherits from ListCreate) instead of `NodeBibliographicContributorsList` because of issues caused by Preprints not being a descendant of Node (Registrations inherit from `NodeBibliographicContributorsList` well because they are also Nodes).